### ISSUE TITLE
Duplicate intermediate certificate in result of gandi certificate export --intermediate something

### ISSUE THREAD

**Issue by aegiap** [Gandi/gandi.cli#201]
I downloaded a SHA2 generated certificate with the Gandi CLI:

```
gandi certificate export service.domain.tld --intermediate
```

And I got the _service.domain.tld.crt_ which is ok and a _service.domain.tld.inter.crt_ which contains all the intermediates certificates for a valid chain. But the file _service.domain.tld.inter.crt_ contains twice the laste certificate.

```
$ grep -B1 'END CERTIFICATE' service.domain.tld.inter.crt 
BT02Vf6Dsuimrdfp5gJ0iHRc2jTbkNJtUQoj1iM=
-----END CERTIFICATE-----
--
0fKtirOMxyHNwu8=
-----END CERTIFICATE-----
--
0fKtirOMxyHNwu8=
-----END CERTIFICATE-----
```

And if I split _service.domain.tld.inter.crt_ in one file per certificate:

```
$ sha1sum intermediate*
fc1afa9e598f3fdc7d606e814f11c0613f210c35  intermediate1.crt
9cb47ccea0732c6f0d1aa2476778a914d985e3a3  intermediate2.crt
9cb47ccea0732c6f0d1aa2476778a914d985e3a3  intermediate3.crt
```

The intermediate concatenate file should contain only two intermediate files.

Reproduced with:

```
ii python-gandicli 0.19
```

**Comment by Simakink** [comment]
+1

**Comment by sayoun** [comment]
Will be fixed in next release 0.20
